libs/core/src/components/sage-link/test/sage-link.e2e.ts

```diff
@@ -0,0 +1,37 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('sage-link', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<sage-link></sage-link>');
+    const element = await page.find('sage-link');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('renders changes to the href', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<sage-link href="#some-bookmark-link"></sage-link>');
+    const component = await page.find('sage-link');
+    expect(component).toHaveClass('hydrated');
+
+    const element = await page.find('sage-link >>> a');
+    expect(element.textContent).toEqual(`#some-bookmark-link`);
+
+    component.setProperty('href', '#my-new-bookmark');
+    await page.waitForChanges();
+    expect(element.textContent).toEqual(`#my-new-bookmark`);
```

Comment:
Does this test the content of the `href=""`?

Reply:
Resolved via the conversation with Julian and Chris